packages/components/src/buttons/tests/chromatic/Button.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { Div } from "@hopper-ui/styled-system";
 import type { Meta, StoryObj } from "@storybook/react";
 import { within } from "@storybook/test";
 
-import { IconList } from "../../../IconList/src/IconList.tsx";
+import { IconList } from "../../../IconList/index.ts";
```

Comment:
well I saw this error when I was doing my chromatic tests. I had copied it from button. Since I fixed it on my component, I fixed it everywhere else. It was using a wrong import, but for some reason, it wasn't displaying the error here, very weird. It's so small, I didn't want to create a whole other pr.